packages/nimble-components/.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I tried deleting the config file because it was essentially empty after removing both presets. No compile errors and the stories still render fine, so I went with that approach and avoided configuring babel at all.

Reply:
I tried deleting the config file because it was essentially empty after removing both presets. No compile errors and the stories still render fine, so I went with that approach and avoided configuring babel at all.

Reply:
Interesting, the migration guide cites it as required. Maybe the note about react-webpack5 applies to html-webpack5 as well 🤷‍♂️
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#babel-mode-v7-exclusively
If it works out works 😂